.claude/commands/add-enhancement.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Create a new OpenShift Enhancement Proposal
+args:
+  - name: area
+    description: Enhancement area (subdirectory under enhancements/)
+  - name: name
+    description: One-line title describing the enhancement
+  - name: description
+    description: Detailed description (what, why, who)
+---
+
+You are tasked with creating a new OpenShift Enhancement Proposal based on the template at `guidelines/enhancement_template.md`.
+
+## Inputs Provided
+
+- **Area**: {{area}}
+- **Name**: {{name}}
+- **Description**: {{description}}
+
+## Instructions
+
+Act as an experienced software architect to create a comprehensive enhancement proposal. Follow these steps:
```

Comment:
Hm, I wonder whether this "Act as an experienced software architect" phrase contributes to the confidently wrong nature of the output.  Maybe, "Act as a competent, experienced software architect with impostor syndrome"?  (Ha ha, only serious.)

Reply:
so, my worry here is that whatever we add as a context will be the used context, so adding something like impostor syndrome will just add wrong things as a question instead of as a conclusion :D and I know exactly how it is!